SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+If you have discovered a security vulnerability in this project, please report it
+privately. **Do not disclose it as a public issue.** This gives us time to work with you
+to fix the issue before public exposure, reducing the chance that the exploit will be
+used before a patch is released.
+
+You may submit the report in the following ways:
+
+- send an email to ???@???; and/or
+- send a [private vulnerability report](https://github.com/keras-team/keras-cv/security/advisories/new)
```

Comment:
I would actually just vote for just this latter option here. Seems simpler, and that we don't have to maintain a separate email address for this.

Is there a downside?

Reply:
@pnacht as to the question about the right mental model for `keras-cv` and `keras-nlp`, they can be thought of as equivalent to `keras-team/keras` at least from this perspective. Maintained by the Keras team directly, supporting TF but not under the TF umbrella for maintenance/contribution.

Reply:
Enabled the reporting now. Thanks!